### PR TITLE
UX-1574/ trimmed trailing whitespaces in Input

### PR DESCRIPTION
**Pull request by patil-pratik-87**
https://user-images.githubusercontent.com/63217589/163549485-9f88488c-6d68-4e7d-8aca-3cabe2298528.mov

### CODE REVIEW

**Comment by patil-pratik-87** [comment]
Thanks @jruddell  for the tip 👍🏽

File: src/app/core/elements/input/Input.tsx [hunk 1]
```diff
@@ -65,6 +65,15 @@ const Input = ({
     [onChange],
   )
 
+  const handleFocusOut = useCallback(
+    (event) => {
+      event.target.value = event?.target?.value?.trim()
```
Comment (jruddell) [inline src/app/core/elements/input/Input.tsx:70]: I'm really not a fan of this. We should think about changing this model to surface out the value only and or the event as a second param

File: src/app/core/elements/input/Input.tsx [hunk 2]
```diff
@@ -65,6 +65,15 @@ const Input = ({
...
+      onChange(event)
+      toggleFocused()
+    },
+    [toggleFocused],
+  )
+
   const handleFocusToggle = useCallback(() => {
```
Comment (jruddell) [inline src/app/core/elements/input/Input.tsx:77]: is this fn still used?

**Review by jruddell — APPROVED** [review]

File: src/app/core/elements/input/Input.tsx (hunk 2 above)
Comment (patil-pratik-87) [reply src/app/core/elements/input/Input.tsx:77]: Yeah, its a `onFocus` handler

File: src/app/core/elements/input/Input.tsx (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/core/elements/input/Input.tsx:70]: yeah, did true, did not wanna make a lot of changes at this point to pass in just the value for the the fear of regression, hence this...

**Pull request merged**